src/unsplashClient.js

```diff
@@ -56,6 +57,22 @@ export default class UnsplashClient {
     };
   }
 
+  /**
+   * dynamic image resizing
```

Comment:
What do you think?

```suggestion
   * Applies image resize parameters on the given URL.
```

Reply:
I think it is a good change, I have already applied the suggestion.